CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
   feature added complexity and browser issues and the use case is likely
   handled by semantic versioning and using a proper dependency.
 
+## 0.5.x
+- Do not use native types to create IRIs in value expansion.
+
```

Comment:
We'll want to make this follow our convention. @davidlehn -- any tips/links to point out?

Reply:
Added similar comment here: https://github.com/digitalbazaar/jsonld.js/pull/210/files#r160245707
I'll fix it for next 0.5.x release.